portfolio/src/main/webapp/script.js

```diff
@@ -14,7 +14,7 @@
 
 document.addEventListener('DOMContentLoaded',function(event){
   // array with texts to type in typewriter
-  var dataText = [ "Programmer", "Learner" , "Dancer"];
+  var dataText = [ "Programmer", "Learner" , "Dancer"]; 
```

Comment:
Weird whitespace at the end, consider running an autoformatter on your code before submitting. I believe Lim Li managed to get one set up with Google formatting rules.